ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py

```diff
@@ -732,6 +732,46 @@ def _squeeze_helper(draw):
     return draw(st.sampled_from(valid_axes))
 
 
+# block
+@st.composite
+def _get_input_and_block(draw):
+    shapes = draw(st.lists(
+        helpers.get_shape(min_num_dims=1, max_num_dims=5, min_dim_size=2, max_dim_size=10),
+        min_size=2, max_size=10))
+    x_dtypes, xs = zip(*[draw(helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=2,
+        max_dim_size=10,
+        shape=shape,
+    )) for shape in shapes])
+    return x_dtypes, xs
+
```

Comment:
I think it is better to have another empty line here due to formatting reasons. Thanks.